intl/msg_hash_us.h

```diff
@@ -6653,11 +6653,11 @@ MSG_HASH(
 
 MSG_HASH(
    MENU_ENUM_LABEL_VALUE_LANG_AFRIKAANS,
-   "Afrikaans"
+   "Afrikaans - Afrikaans"
```

Comment:
I see why you've done this (every other language has a translation, except english) but is it really necessary?

Reply:
@dantob Before my first commit (https://github.com/libretro/RetroArch/pull/10727) It simply pulled the language name from each language, and Arabic (and some characters from Chinese Traditional) was showing up as squares…  ☐☐☐☐☐☐ was not very descriptive for Arabic with the default font…

Reply:
@dantob And for the sake of uniformity, yes, I even translate the language names that are the same in english as the aforementioned language, if that's what you mean (sorry, what you tried to highlight completely flew over my head with my first reply 😂)…